install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -394,10 +398,48 @@ metadata:
 provisioner: kubernetes.io/vsphere-volume <1>
 parameters:
   diskformat: thin <2>
+
 ----
 <1> For more information about using VMWare vSphere with {product-title}, see the link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html[VMWare vSphere documentation].
 <2>  `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. See vSphere docs for details. Default: `thin`
-====
+
+[[azure-unmanaged-disk]]
+=== Azure Unmanaged Disk Object Definition
+
+.azure-unmanaged-disk-storageclass.yaml
+[source,yaml]
+----
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: slow
+  provisioner: kubernetes.io/azure-disk
+  parameters:
+    skuName: Standard_LRS  <1>
+    location: eastus  <2>
+    storageAccount: azure_storage_account_name  <3>
+----
+<1> skuName, Azure storage account Sku tier. Default is empty.
```

Comment:
Please skip the field names from all descriptions. For example change `<1> skuName, Azure storage account Sku tier. Default is empty.` to `<1> Azure storage account Sku tier. Default is empty.`

Reply:
s/Sku/SKU